guides/v3.3.0/object-model/computed-properties.md

```diff
@@ -80,31 +80,68 @@ ironMan.fullName; // no console output since dependencies have not changed
 
 In the previous example, the `fullName` computed property depends on two other properties of the same object.  
 However, you may find that you have to observe the properties of a different object.
+
 For example, look at this computed property:
 
 ```javascript
 import EmberObject, { computed } from '@ember/object';
 
-let obj = EmberObject.extend({
-  baz: { foo: 'BLAMMO', bar: 'BLAZORZ' },
+let home = EmberObject.extend({
+  location: {
+    street: 'Evergreen Terrace',
+    nr: 742
```

Comment:
I much prefer a real example like this over `foo bar`! Thank you! Maybe this object could be `streetName` and `streetNumber` for the keys. That might be clearer to beginners than an abbreviation.